cfpb/cfgov-refresh
# Update Wagtail admin interface for hero block

This updates the labels and help text for the fields that make up the hero block. With this update, we have incorporated designer feedback to bring the hero module in line with the guidelines in the Design Manual, which will make it easier for content owners to create hero graphics that
fit our site.

## Changes

- Update label and help text of several fields in the Hero admin on Wagtail

## Additions

- Migrations to safely update fields so no data is lost and no manual action is needed after deploy

## Testing

1. Pull down this branch
2. Refresh your database and run all migrations
3. Visit some pages that have a hero. Confirm that the text, images, colors, etc. are all displayed as normal. e.g.:
    - http://localhost:8000/policy-compliance/rulemaking/regulations/
    - http://localhost:8000/consumer-tools/auto-loans/
    - http://localhost:8000/data-research/credit-card-data/
4. Log in to wagtail admin and edit any page with a hero header. 
    - The editor should appear as in the screenshot below, with updated field names. It will contain helpful guidelines and embedded links to the Design Manual.

## Screenshots
![sublanding page updated small image help](https://user-images.githubusercontent.com/4295388/61220594-a2776a00-a6e4-11e9-9568-b98461dad6da.png)


## Notes

- Help text (especially links to the Design Manual from the help text) may need to change when our new design system comes online


## Checklist


## Testing checklist

This PR doesn't change any end user-facing pages
